client.js

```diff
@@ -18,90 +18,81 @@ function createClientConfig() {
     }
 }
 
-function mapExample(map) {
-    (async () => {
-        console.log("Now, `map` will be filled with random entries.");
+async function mapExample(map) {
+    console.log("Now, `map` will be filled with random entries.");
 
-        var iterationCounter = 0;
-        while (true) {
-            const randomKey = Math.floor(Math.random() * 100000);
-            await map.put('key' + randomKey, 'value' + randomKey);
-            await map.get('key' + randomKey);
-            if (++iterationCounter === 10) {
-                iterationCounter = 0;
-                map.size().then((size) => console.log(`map size: ${size}`));
-            }
+    let iterationCounter = 0;
+    while (true) {
+        const randomKey = Math.floor(Math.random() * 100000);
+        await map.put('key' + randomKey, 'value' + randomKey);
+        await map.get('key' + randomKey);
+        if (++iterationCounter === 10) {
+            iterationCounter = 0;
+            map.size().then((size) => console.log(`map size: ${size}`));
```

Comment:
I missed this one in the initial PR, but it would be better if we convert this to async/await syntax as well.

Something like 

```js
const size = await map.size();
console.log(`map size: ${size}`);
```

Same for the other file

Reply:
Fixed.